divide.py

```diff
@@ -0,0 +1,25 @@
+class IncorrectInput(Exception):
+    pass
+
+
+def divide(numerator=1, denominator=2):
```

Comment:
Perfect testable function! Like how you've given default values however can you see how this could be another good error case to do? As a challenge if you remove these can you throw an error if one or the other doesn't exist?

Reply:
Thanks, I've added a new custom error to catch missing arguments